docs/scripts/generate_examples_tests.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+"""Generates examples/tests.py."""
+
+import os
+from contextlib import redirect_stdout
+from pathlib import Path
+from sys import path
+
+# Override global path to load pyavd from pwd instead of any installed version.
+path.insert(0, str(Path(__file__).parents[2]))
+
+examples_tests_path = Path(__file__).parents[2] / "examples" / "tests.yaml"
+
+
+prev = os.environ.get("TERM", "")
+os.environ["TERM"] = "dumb"
+# imported after TERM is set to act upon rich console.
+from anta.cli.get.commands import tests  # noqa: E402
```

Comment:
Are we ok with using click functions in scripts?

Reply:
Hmm I would say yes - people developing should have click installed - thoughts?

Reply:
Maybe we should have a generic function called `get_tests` or something that we can use here and in the `tests` click functions.

Reply:
the function would be in `anta.cli.get.utils` and so would import `click` there is not much point I think
This script is used in pre-commit only so thats for us really to generate the doc